src/code42cli/cmds/alert_rules.py

```diff
@@ -68,7 +69,12 @@ def add_user(state, rule_id, username):
 @sdk_options()
 def remove_user(state, rule_id, username):
     """Remove a user from an alert rule."""
-    _remove_user(state.sdk, rule_id, username)
+    try:
+        _remove_user(state.sdk, rule_id, username)
+    except Py42BadRequestError:
```

Comment:
Is there a chance that the rule ID or username could somehow cause a bad request (from the CLI) for other reasons?

Reply:
I just tested these cases and they both provide correct error messages.